projects/magma/Dockerfile

```diff
@@ -0,0 +1,98 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+FROM gcr.io/oss-fuzz-base/base-builder
+RUN apt-get update && apt-get -y upgrade
+RUN git clone --depth 1 https://github.com/0x34d/magma.git
+#RUN git clone --depth 1 https://github.com/magma/magma.git
+COPY build.sh $SRC/build.sh
+COPY build_defs.bzl.patch $SRC/build_defs.bzl.patch
+WORKDIR $SRC/magma/
+
+
+
+RUN apt-get update && \
+  # Setup necessary tools for adding the Magma repository
+  apt-get install -y apt-utils software-properties-common apt-transport-https gnupg wget && \
+  # Download Bazel
+  wget -P /usr/sbin --progress=dot:giga https://github.com/bazelbuild/bazelisk/releases/download/v1.10.0/bazelisk-linux-amd64 && \
+  chmod +x /usr/sbin/bazelisk-linux-amd64 && \
+  ln -s /usr/sbin/bazelisk-linux-amd64 /usr/sbin/bazel
```

Comment:
We already install bazel, is the version we have not working?

Reply:
Kind off.

Reply:
only `5.2.0` is allowed

Reply:
Done in the next Push.